client/extensions/wp-job-manager/state/settings/test/actions.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+/**
+ * Internal dependencies
+ */
+import {
+	WP_JOB_MANAGER_FETCH_ERROR,
+	WP_JOB_MANAGER_FETCH_SETTINGS,
+	WP_JOB_MANAGER_UPDATE_SETTINGS,
+} from '../../action-types';
+import {
+	fetchError,
+	fetchSettings,
+	updateSettings,
+} from '../actions';
+
+describe( 'actions', () => {
+	const siteId = 123456;
+	const settings = {
+		data: {
+			job_manager_per_page: 25,
+			job_manager_hide_filled_positions: true,
+		}
+	};
+
+	describe( '#updateSettings()', () => {
+		it( 'should return an action object', () => {
+			const action = updateSettings( siteId, settings.data );
+
+			expect( action ).to.eql( {
```

Comment:
Personally, I prefer `deep.equal` since it's less prone to be confused with `equal`, but that's a matter of taste, I guess :slightly_smiling_face: